who's_your_daddy4_3.py

```diff
@@ -0,0 +1,30 @@
+dict = {
+    "Florin Piersic": 'Florin Piersic JR',
+    "Douglas": "Douglas JR",
+    "Denzel Washinghton": "John Washington",
+    "Stellan Skarsgard": "Alexander Skarsgard",
+    "Quincy Jones": "Rashida Jones"
+}
+
+
+print("Enter the name of actor: ")
+father = input()
+print (dict.get(father))
+print("What do you want to do? Change or Delete")
+mod = input()
+
+print("Is this correct?(Yes/NO)")
+bool = input()
+if bool == "Yes":
+    print ("Cool")
+else:
+    print("Change the name of the son?(Yes/NO)")
+    a = input()
+    if a == "Yes":
+        dict["Florin Piersic"] = input("Enter the ", father, "son")
+        print("The son of", father, "is :", dict.get(father))
+    else:
+        print("Cool")
+
+
```

Comment:
I do not understand why "bool" and 'a' variables are needed. From my perspective we do not need those.

Reply:
Dictionary is son-father, in the above case is father-son